mmv1/products/firebasehosting/Version.yaml

```diff
@@ -46,6 +46,13 @@ examples:
       site_id: 'site-id'
     test_env_vars:
       project_id: 'PROJECT_NAME'
+  - name: 'firebasehosting_version_headers'
```

Comment:
Can you add another test example to cover the missing test for `regex` field?

Reply:
Okay.